framework/tlvf/AutoGenerated/src/tlvf/ieee_1905_1/tlvDeviceInformation.cpp

```diff
@@ -37,59 +37,83 @@ sMacAddr& tlvDeviceInformation::mac() {
     return (sMacAddr&)(*m_mac);
 }
 
-uint8_t& tlvDeviceInformation::info_length() {
-    return (uint8_t&)(*m_info_length);
+uint8_t& tlvDeviceInformation::local_interface_list_length() {
+    return (uint8_t&)(*m_local_interface_list_length);
 }
 
-std::tuple<bool, tlvDeviceInformation::sInfo&> tlvDeviceInformation::info(size_t idx) {
-    bool ret_success = ( (m_info_idx__ > 0) && (m_info_idx__ > idx) );
+std::tuple<bool, cLocalInterfaceInfo&> tlvDeviceInformation::local_interface_list(size_t idx) {
+    bool ret_success = ( (m_local_interface_list_idx__ > 0) && (m_local_interface_list_idx__ > idx) );
```

Comment:
What's the purpose of the `m_local_interface_list_idx__` variable? Doesn't it make more sense to use `m_local_interface_list.size()`:
`bool ret_success = idx >=0 && idx < m_local_interface_list.size()` ?